samples/nrf9160/lwm2m_client/src/ipso/ipso_buzzer.c

```diff
@@ -0,0 +1,263 @@
+/*
+ * Copyright (c) 2019 Foundries.io
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/*
+ * Source material for IPSO Buzzer object (3338):
+ * http://www.openmobilealliance.org/tech/profiles/lwm2m/3338.xml
+ */
+
+/*
+ * This file exists due to an error in zephyrs handling of receiving floats over lwm2m.
```

Comment:
Float fixes are already on a PR (https://github.com/zephyrproject-rtos/zephyr/pull/37146) so it should be integrated soon. I'd rather document the known issue for now instead of duplicating object implementation.

Reply:
Fixed